src/StatusQ/Controls/StatusBaseButton.qml

```diff
@@ -18,159 +18,119 @@ Rectangle {
         Danger
     }
 
-    property StatusAssetSettings icon: StatusAssetSettings {
-        width: 24
-        height: 24
-    }
+    property StatusAssetSettings asset: StatusAssetSettings { }
 
     property bool loading: false
 
-    property alias hovered: sensor.containsMouse
+    property color normalColor
+    property color hoverColor
+    property color disaledColor
+
+    property color textColor
+    property color disabledTextColor
+    property color borderColor: "transparent"
 
     property int size: StatusBaseButton.Size.Large
     property int type: StatusBaseButton.Type.Normal
 
-    property alias text: label.text
-    property alias font: label.font
-
     property real defaultLeftPadding: size === StatusBaseButton.Size.Large ? 24 : 12
     property real defaultRightPadding: size === StatusBaseButton.Size.Large ? 24 : 12
     property real defaultTopPadding: {
         switch (size) {
-            case StatusBaseButton.Size.Tiny:
-              return 5
-            case StatusBaseButton.Size.Small:
-              return 10
-            case StatusBaseButton.Size.Large:
-            default:
-              return 11
+        case StatusBaseButton.Size.Tiny:
+            return 5
+        case StatusBaseButton.Size.Small:
+            return 10
+        case StatusBaseButton.Size.Large:
+        default:
+            return 11
         }
     }
     property real defaultBottomPadding: {
         switch (size) {
-            case StatusBaseButton.Size.Tiny:
-              return 5
-            case StatusBaseButton.Size.Small:
-              return 10
-            case StatusBaseButton.Size.Large:
-            default:
-              return 11
+        case StatusBaseButton.Size.Tiny:
+            return 5
+        case StatusBaseButton.Size.Small:
+            return 10
+        case StatusBaseButton.Size.Large:
+        default:
+            return 11
```

Comment:
This could really just `return defaultTopPadding` as they're the same :)

Reply:
done.